tests/OmniSharp.Roslyn.CSharp.Tests/FindSymbolsFacts.cs

```diff
@@ -161,7 +161,7 @@ public async Task Returns_struct_kind(string filename)
 
             var usages = await FindSymbolsAsync(code, filename);
             var symbols = usages.QuickFixes.Cast<SymbolLocation>().Select(q => q.Kind);
-            Assert.Equal("Structure", symbols.First());
+            Assert.Contains(symbols.First(), new[] { "Struct", "Structure" });
```

Comment:
In the TypeKind enum both `Struct` and `Structure` have the same value. A runtime change must have effected the order they are returned. Either is valid since they both represent the value `10`.